trade-finance-manager-ui/templates/utilisation-reports/record-corrections/check-the-information.njk

```diff
@@ -107,7 +107,7 @@
         href: cancelLink,
         classes: "govuk-button--secondary",
         attributes: {
-          'data-cy': 'cancel-button'
+          'data-cy': 'cancel-link'
```

Comment:
is this a button or a link on the page? given the class is still govuk-button?

Reply:
I think we should keep this as `button` - I think `link` makes sense for the other pages whose 'Cancel' behaviour is through a hyperlink

Reply:
I named it this just to use the common cypress selector, have seen Matt has added a new one for cancel button so will revert to that change when I merge in main.